Week1/homework/world-queries.txt

```diff
@@ -37,4 +37,190 @@ SELECT city.Name FROM city JOIN country ON city.CountryCode = country.Code WHERE
+| Saint Kitts and Nevis            |
+| Saint Lucia                      |
+| Mexico                           |
+| Montserrat                       |
+| Martinique                       |
+| Nicaragua                        |
+| Panama                           |
+| Puerto Rico                      |
+| El Salvador                      |
+| Saint Pierre and Miquelon        |
+| Turks and Caicos Islands         |
+| Trinidad and Tobago              |
+| United States                    |
+| Saint Vincent and the Grenadines |
+| Virgin Islands, British          |
+| Virgin Islands, U.S.             |
++----------------------------------+
+37 rows in set (0.00 sec)
+
+4- What's the top 10 countries by SurfaceArea ?
+
+SELECT name, surfacearea FROM country ORDER BY surfacearea DESC limit 10;
++--------------------+-------------+
+| name               | surfacearea |
++--------------------+-------------+
+| Russian Federation | 17075400.00 |
+| Antarctica         | 13120000.00 |
+| Canada             |  9970610.00 |
+| China              |  9572900.00 |
+| United States      |  9363520.00 |
+| Brazil             |  8547403.00 |
+| Australia          |  7741220.00 |
+| India              |  3287263.00 |
+| Argentina          |  2780400.00 |
+| Kazakstan          |  2724900.00 |
++--------------------+-------------+
+10 rows in set (0.01 sec)
+
+5- What's the top 10 most populated cities?
+
+SELECT name, population FROM city ORDER BY population DESC limit 10;
++-----------------------+------------+
+| name                  | population |
++-----------------------+------------+
+| Mumbai (Bombay)       |   10500000 |
+| Seoul                 |    9981619 |
+| Sï¿½o Paulo           |    9968485 |
+| Shanghai              |    9696300 |
+| Jakarta               |    9604900 |
+| Karachi               |    9269265 |
+| Istanbul              |    8787958 |
+| Ciudad de Mï¿½xico    |    8591309 |
+| Moscow                |    8389200 |
+| New York              |    8008278 |
++-----------------------+------------+
+10 rows in set (0.01 sec)
+
+6- Select the top 3 countries by population that their names start with the letter ‘P’
+
+SELECT name, population FROM country where name LIKE 'P%'  ORDER BY population DESC limit 3;
++-------------+------------+
+| name        | population |
++-------------+------------+
+| Pakistan    |  156483000 |
+| Philippines |   75967000 |
+| Poland      |   38653600 |
++-------------+------------+
+3 rows in set (0.01 sec)
+
+7- What is the population in Asia?
+
+SELECT continent, sum(population) FROM country WHERE continent = 'Asia';
++-----------+-----------------+
+| continent | sum(population) |
++-----------+-----------------+
+| Asia      |      3705025700 |
++-----------+-----------------+
+1 row in set (0.00 sec)
+
+8- Which languages are spoken in the Region of ‘South America’?
+
+SELECT language FROM countrylanguage JOIN country ON countrylanguage.countrycode = country.code where continent = 'South America';
```

Comment:
The country table contains a "region" column that you can use instead of the "continent" column in the where clause

Reply:
I think region has different meaning than continent, didn't you mean "Which languages are spoken in the Continent of ‘South America’?", i think South America is a continent not a region